docs/authoring-tasks.adoc

```diff
@@ -47,7 +47,7 @@ The pipeline generated by ODS is automatically fed with a few parameters which c
 
 Pipelines may produce and consume artifacts such as xUnit test results, image digests, static code analysis reports, etc.
 
-These artifacts are stored under `.ods/artifacts`, for example `.ods/artifacts/xunit-reports/report.xml`. The `ods-finish` task at the end of each pipeline automatically uploads all artifacts stored under `.ods/artifacts` to Nexus. Conversly, the `ods-start` task searches for any artifacts in Nexus belonging to the checked out commit (this may happen e.g. if the pipeline was triggered again or the same commit is pushed to another branch). If artifacts are detected, they will be downloaded and subsequent tasks may change their behaviour dependening on the existance of such artifacts. For example, a task might skip running the testsuite if an xUnit report already exists.
+These artifacts are stored under `.ods/artifacts`, for example `.ods/artifacts/xunit-reports/report.xml`. The `ods-finish` task at the end of each pipeline automatically uploads all artifacts stored under `.ods/artifacts` to Nexus. Conversly, the `ods-start` task searches for any artifacts in Nexus belonging to the checked out commit (this may happen e.g. if the pipeline was triggered again or the same commit is pushed to another branch). If artifacts are detected, they will be downloaded and subsequent tasks may change their behaviour dependening on the existance of such artifacts.
```

Comment:
Conversely

Reply:
```suggestion
These artifacts are stored under `.ods/artifacts`, for example `.ods/artifacts/xunit-reports/report.xml`. The `ods-finish` task at the end of each pipeline automatically uploads all artifacts stored under `.ods/artifacts` to Nexus. Conversely, the `ods-start` task searches for any artifacts in Nexus belonging to the checked out commit (this may happen e.g. if the pipeline was triggered again or the same commit is pushed to another branch). If artifacts are detected, they will be downloaded and subsequent tasks may change their behaviour dependening on the existance of such artifacts.
```